aws/resource_aws_launch_template.go

```diff
@@ -1211,6 +1239,17 @@ func buildLaunchTemplateData(d *schema.ResourceData) (*ec2.RequestLaunchTemplate
 		}
 	}
 
+	if v, ok := d.GetOk("hibernation_options"); ok {
```

Comment:
Refactor to `expandLaunchTemplateInstanceHibernationOptions`?

Reply:
done